access/ms-teams/app.go

```diff
@@ -0,0 +1,517 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"google.golang.org/grpc"
+	grpcbackoff "google.golang.org/grpc/backoff"
+
+	"github.com/gravitational/teleport-plugins/lib"
+	"github.com/gravitational/teleport-plugins/lib/logger"
+	pd "github.com/gravitational/teleport-plugins/lib/plugindata"
+	"github.com/gravitational/teleport-plugins/lib/stringset"
+	"github.com/gravitational/teleport-plugins/lib/watcherjob"
+	"github.com/gravitational/teleport/api/client"
+	"github.com/gravitational/teleport/api/client/proto"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/trace"
+)
+
+const (
+	// pluginName used as Teleport plugin identifier
+	pluginName = "ms-teams"
+	// minServerVersion is the minimal teleport version the plugin supports.
+	minServerVersion = "6.1.0-beta.1"
```

Comment:
Where did this version come from? Should probably set it to something that is supported. How about `8.0.0`?

Reply:
Frankly, I do not know. Looks like that minimal version was copy pasted all over the plugins from the beginning of times. I've replaced it here, but I guess we need to create an issue to replace it everywhere else.